syntax = "proto3";

package order;

service OrderService {
  rpc PlaceOrder (OrderRequest) returns (OrderResponse);
  rpc UpdateOrder (OrderRequest) returns (OrderResponse);
}

message OrderRequest {
  int32 product_id = 1;
  int32 order_id = 2;
  int32 quantity = 3;
}

message OrderResponse {
  int32 order_id = 1;
}
